source/docs/advanced/password/configure-password.md

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
I am unsure whether this is the good location for this topic or not.

Any idea ?

Reply:
moved into features